lib/main_common.pm

```diff
@@ -1262,6 +1262,7 @@ sub load_x11tests {
         loadtest "x11/midori" unless (is_staging || is_livesystem);
         loadtest "x11/ristretto";
     }
+    loadtest "x11/tomcat" unless is_sle('<12-sp2');
```

Comment:
this way you will also trigger it for SLE 12 SP5 and SLE 15 SP2 . Maybe it is good but need to check this or limit execution to already released versions

Reply:
I think it is ok if it is triggered for those SLE versions as well, I would keep close attention to the runs on the first couple of weeks in order to fix any potential issues and make the test more robust.

Reply:
Why schedule it here ? allpatterns is alredy long enough (1:30), maybe better put it into webserver or extratests ?

Reply:
This will affect also staging

Reply:
Personally, I would go for `EXTRATEST=tomcat`

Reply:
> Personally, I would go for `EXTRATEST=tomcat`

so, you mean create a new extra_tests test suite for tomcat?

Reply:
Yes, in the light of current trends (splitting test suites), I would create `load_extra_tests_tomcat` and put it there.